### ISSUE TITLE
support geojson polygons

### ISSUE THREAD

**Issue by contra** [neumino/thinky#486]
These are valid RethinkDB types. It seems like the documentation hints that it is something that should be done eventually, but there is no ticket open for it.

<img width="590" alt="screen shot 2016-04-25 at 2 37 17 pm" src="https://cloud.githubusercontent.com/assets/425716/14794751/6aa5178e-0af3-11e6-88bf-a05584920121.png">

**Comment by neumino** [comment]
Yep, you can't use a polygon in a schema. You can use it to filter things though.

**Comment by contra** [comment]
https://rethinkdb.com/api/javascript/geojson/ shows an example of converting GeoJSON to a ReQL Geometry object, which you can filter by AFAIK. geojson types and geojson validation are supported first class in the database, Point objects are just one of those geojson types.

**Comment by deoqc** [comment]
+1
using `type.any()` for now
